tools/importer/cmsis_importer.json

```diff
@@ -0,0 +1,152 @@
+{
+  "config" : {
+    "cmsis_repo" : "https://github.com/ARM-software/CMSIS_5",
+    "cmsis_branch" : "develop"
+  },
+  "files" : [
+    {
+      "cmsis_file" : "CMSIS/Core/Template/ARMv8-M/tz_context.c",
```

Comment:
This is both extremely verbose and slow. Instead this should be a map from `cmsis_file` to `mbed_file`.

For example:
```json
"files" : {
    "CMSIS/Core/Template/ARMv8-M/tz_context.c": "platform/mbed_tz_context.c"
    ...
}
```